frontend/src/app/modules/common/dynamic-forms/components/dynamic-field-wrapper/dynamic-field-wrapper.component.html

```diff
@@ -1,7 +1,7 @@
 <op-form-field [label]="to?.label"
                [noWrapLabel]="to?.noWrapLabel"
                [required]="to?.required"
-               [helpTextAttribute]="field.key"
+               [helpTextAttribute]="to?.property"
```

Comment:
FYI @b12f , property is the api key name (such as "status", "customField1") while the key is the model key for formly, which can be "_links.customField1" and breaks the attribute help text)